src/input_context/actions.rs

```diff
@@ -6,9 +6,9 @@ use core::{
     fmt::{self, Debug, Display, Formatter},
     marker::PhantomData,
 };
+use log::debug;
 
 use bevy::{platform::collections::hash_map::Entry, prelude::*, utils::TypeIdMap};
-use log::debug;
```

Comment:
Please, align the log as before. Can't make suggestions since git thinks it's a deleted line.

Reply:
oops!